Shortest-Path-Finder (GUI)/Shortest Path Finder.py

```diff
@@ -0,0 +1,252 @@
+# Click on boxes to mark start and end point. Then make some obstacles and once done
+# press SPACE to find the shortest path between the start and end point
+
+import pygame as pg
+import math
+from queue import PriorityQueue
+
+
+WIDTH = 600
+WIN = pg.display.set_mode((WIDTH, WIDTH))

```

Comment:
Change the width to the multiple of your rows(e.g. change the width to 595)

Reply:
> Change the width to the multiple of your rows(e.g. change the width to 595)

Sure I'll do it ASAP. Thanks for the suggestion.